### PR TITLE
first opensource commit

### PR DESCRIPTION
**Pull request by archanadubey**
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/osci_2023/archanaDu.markdown [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Archana 
+short_name: archi
+
+github: archanadubey
```
Comment (joshuarrrr) [inline _authors/osci_2023/archanaDu.markdown:5]: The formatting of this frontmatter section is important:
```suggestion
github: archanadubey
---
```

File: _authors/osci_2023/archanaDu.markdown [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+**Archana** have done bachelor and masters but I have a long career gap. I am again trying to reform my career in software enginnering as a java developer. Looking forawrd to get support and guidence to achive my goal. there are a lot of road blocks in path. Hope opensource  will experience will clear blocks and I will have smooth journey afterward.
```
Comment (joshuarrrr) [inline _authors/osci_2023/archanaDu.markdown:7]: Some spelling and typo suggestions:
```suggestion
**Archana** have done bachelor and masters but I have a long career gap. I am again trying to reform my career in software engineering as a java developer. Looking forward to get support and guidance to achieve my goal. There are a lot of road blocks in the path. I hope opensource experience will clear blocks and I will have smooth journey afterward.
```

**Review by joshuarrrr — CHANGES_REQUESTED** [review]
Needs some formatting updates, but otherwise looks good.

**Comment by dtaivpp** [comment]
@archanadubey please address Josh's comments and we should be good, thank you!

**Comment by dtaivpp** [comment]
Closing as we are past deadline and there are outstanding changes.

**Pull request closed without merging**